framework/wazuh/core/config/models/server.py

```diff
@@ -376,7 +376,7 @@ class ServerConfig(WazuhConfigBaseModel):
     communications: CommunicationsConfig = CommunicationsConfig()
     logging: LoggingConfig = LoggingConfig(level=LoggingLevel.info)
     cti: CTIConfig = CTIConfig()
-    jwt: JWTConfig
+    jwt: JWTConfig = JWTConfig()
```

Comment:
Instead of naming it `jwt` I would prefer `server.private_key` since it's also used to generate the certificates

```suggestion
    private_key: str = ''
```

Reply:
The following issue was created to make these changes https://github.com/wazuh/wazuh/issues/28415.